### PR TITLE
🎨 Frontend: API Keys enhancements

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Support expiration date when creating keys
- [x] Partially hide key and secret

APIKeys

## Related issue/s

closes https://github.com/ITISFoundation/osparc-simcore/issues/4541
closes https://github.com/ITISFoundation/osparc-simcore/issues/5658

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by wvangeit — APPROVED** [review]
Great enhancement, thanks

**Review by sanderegg — APPROVED** [review]
great! thanks

File: services/static-webserver/client/source/class/osparc/desktop/preferences/pages/TokensPage.js [hunk 1]
```diff
@@ -73,12 +73,17 @@ qx.Class.define("osparc.desktop.preferences.pages.TokensPage", {
...
+      createAPIKeyWindow.addListener("finished", e => {
+        const formData = e.getData();
         const params = {
           data: {
-            "display_name": keyLabel.getData()
+            "display_name": formData["name"]
           }
         };
+        if (formData["expiration"]) {
+          const seconds = parseInt((new Date(formData["expiration"]).getTime() - new Date().getTime()) / 1000);
```
Comment (wvangeit) [inline services/static-webserver/client/source/class/osparc/desktop/preferences/pages/TokensPage.js:84]: Just a small question here. This is running client-side, right? But wouldn't these seconds be used server-side? There might be a time-zone issue? Or not?

Comment (wvangeit) [reply services/static-webserver/client/source/class/osparc/desktop/preferences/pages/TokensPage.js:84]: Ok, crazy question, please ignore. It's seconds, won't matter :-)

Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/preferences/pages/TokensPage.js:84]: Everything happens in the frontend's (user's) time zone: the date selection and more importantly, the delta calculation. Furthermore, the web api is already expecting the expiration in seconds.

**Review by ignapas — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**